### PR TITLE
[React, Angular, Vue & Layout] Techs Refactor 

### PR DESCRIPTION
**Pull request by Dorian30**
## Summary

- Added suggested structure in every tech inside Front-End tree folder. 

``` 
- Project Bootstrapper
- Style Guide
- Internal Libraries
- External Libraries
```

- Place the content of each of the sections above inside its corresponding tech main page. 
- Deleted unused content.
- Fixed links and typo errors.

### React 

Fix

### Vue 

image

### Angular 

image

### AngularJS

image

### Layout

image

## Trello Card

https://trello.com/c/FU10egEJ/10-refactor-cada-tech

### CODE REVIEW

File: frontend/angular-js/README.md [hunk 1]
```diff
@@ -1,17 +1,26 @@
 # AngularJS Tech Guides [Tech Guides under construction]
 
-## Kickoff Guide
+## Content
 
-At [Wolox](http://wolox.com.ar), we have developed the [Angular kickoff guide](./docs/kickoff/README.md) to create our Angular projects. Check it out to find a detailed guide to start with your project.
+- [Project Bootstrapper](#project-bootstrapper)
+- [Style Guide](#style-guide)
+- [Open Source](#open-source)
+- [Useful Documentation](#useful-documentation)
```
Comment (SKOLZ) [inline frontend/angular-js/README.md]: I'd delete the content of this section for now. just leave the title.

Comment (SKOLZ) [reply frontend/angular-js/README.md]: add internal and external libraries like in react but with empty content for now.

File: frontend/angular/README.md [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
-## Recommended Libraries
+## Content
 
-- [Internal Libraries]
+- [Project Bootstrapper](#project-bootstrapper)
+- [Style Guide](#style-guide)
+- [Internal Libraries](#internal-libraries)
+- [External Libraries](#external-libraries)
+- [Useful Documentation](#useful-documentation)
+- [Posts](#posts)
```
Comment (SKOLZ) [inline frontend/angular/README.md]: delete this, all angular posts should be in the front-end readme with the prefix `[Angular]`

File: frontend/angular/README.md [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
 
-- [External Libraries]
+## Project Bootstrapper
 
-## Useful Documentation
+## Style Guide
+
+## Internal Libraries
```
Comment (SKOLZ) [inline frontend/angular/README.md]: add empty table here

File: frontend/angular/README.md [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
 
-- [Style Guide]
+## External Libraries
```
Comment (SKOLZ) [inline frontend/angular/README.md]: add empty table here

File: frontend/angular-js/README.md [hunk 5]
```diff
@@ -1,17 +1,26 @@
...
+At [Wolox](http://wolox.com.ar), we have developed the [Angular Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch) to create our Angular projects. You'll find a detailed guide to start with your project in its README.
+
+## Style Guide
+
+We use the following [Style Guide](https://github.com/johnpapa/angular-styleguide) as reference for our Angular style projects.
 
 ## Open Source
 
-- [Wolox Front-End Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch)
 - [Custom Multiple Select](https://codepen.io/SKOLZ/pen/PJqPGG)
```
Comment (SKOLZ) [inline frontend/angular-js/README.md]: move this to the front-end readme.

File: frontend/layout/README.md [hunk 6]
```diff
@@ -1,16 +1,21 @@
 # Layout Tech Guides [Tech Guides under construction]
 
-## Kickoff Guide
+## Content
 
-At [Wolox](http://wolox.com.ar), we have developed the [Layout kickoff guide](../docs/kickoff/README.md) to create our layout projects. Check it out to find a detailed guide to start with your project.
+- [Project Bootstrapper](#project-bootstrapper)
+- [Style Guide](#style-guide)
+- [Open Source](#open-source)
```
Comment (SKOLZ) [inline frontend/layout/README.md]: internal and external libraries with empty tables instead

File: frontend/layout/README.md [hunk 7]
```diff
@@ -1,16 +1,21 @@
...
+
+## Project Bootstrapper
+
+At [Wolox](http://wolox.com.ar), we have developed the [Front-End Bootstrap](http://github.com/wolox/frontend-bootstrap) to create our React projects. You'll find a detailed guide to start with your project in its README.
 
 ## Style Conventions
```
Comment (SKOLZ) [inline frontend/layout/README.md]: style guide

File: frontend/vue/README.md [hunk 8]
```diff
@@ -1 +1,23 @@
...
+This section is still under construction. 
+
+## Content
+
+- [Project Bootstrapper](#project-bootstrapper)
+- [Style Guide](#style-guide)
+- [Internal Libraries](#internal-libraries)
+- [External Libraries](#external-libraries)
+- [Useful Documentation](#useful-documentation)
+- [Posts](#posts)
```
Comment (SKOLZ) [inline frontend/vue/README.md]: delete this

File: frontend/angular-js/README.md (hunk 5 above)
Comment (Dorian30) [reply frontend/angular-js/README.md]: You mean to add an `open source` section in the front end main page ?

**Pull request merged**